### ISSUE TITLE
Materials and Goo after 1.4.1

### ISSUE THREAD

**Issue by irineus** [confusingbits/ForScience#7]
Hello.

It seems there's a problem with "SC-9001 Science Jr" and "Mystery Goo™ Containment Unit" science readings after 1.4.1 (at 1.4 it got the same problem). You need to go EVA and reset it manually in order to grab the science.

For other parts, seems works fine in my tests:
- Double-C Seismic Accelerometer
- PresMat Barometer
- GRAVMAX Negative Gravioli Detector
- 2HOT Thermometer

Didn't made extensive tests on "Atmospheric Fluid Spectro-Variometer", but it seems to work in a couple of flight I've made.

**Comment by Cwhizard** [comment]
Try the cwizard branch.  I don't think this branch is being maintained regularly.  If the issue persists with my branch let me know there and Ill look into it.

**Comment by confusingbits** [comment]
The mod is still being maintained, as it has been for years.

It it usually a mod conflict or installation error that causes an issue. So I do not usually patch unless I get more than one report.

**Comment by irineus** [comment]
So I've made some tests to see if there's any mod conflict. Report and prints below (sorry for the pt-br language in the prints).

1) Removed any other mods:
![mods](https://user-images.githubusercontent.com/32855937/38452396-0a6c7e5e-3a1a-11e8-955c-169ef1713d91.png)

2) Lauched the game (version check below):
![version](https://user-images.githubusercontent.com/32855937/38452404-3198381a-3a1a-11e8-9242-aff0fac016f4.png)

3) Started a new career mode, edited funds and science to upgrade and unlock all science tree, and made a small science rover with all science equip available and with Bob as crew:
![rover](https://user-images.githubusercontent.com/32855937/38452427-8d9acbe6-3a1a-11e8-825f-cfced4d0f37d.png)

4) Launched rover, no science experiments triggered:
![rover launch](https://user-images.githubusercontent.com/32855937/38452452-e7c8e83c-3a1a-11e8-877b-3db983d091ce.png)

5) Moved the rover outside the runway and at the biome change, got science for (in order):
- Crew report
- Atmospheric Fluid Spectro-Variometer
- 2HOT Thermometer
- GRAVMAX Negative Gravioli Detector
- Double-C Seismic Accelerometer
- PresMat Barometer

Got no science for:
- SC-9001 Science Jr
- Mystery Goo™ Containment Unit

![biome change](https://user-images.githubusercontent.com/32855937/38452481-689adb5a-3a1b-11e8-9fee-ab7fccec6d67.png)

6) The only way to grab science for "Mystery Goo™ Containment Unit" is manually trigger it and, of course, manually restore it.
![mistery goo](https://user-images.githubusercontent.com/32855937/38452507-d896718a-3a1b-11e8-8d95-408ac99e9336.png)

So, it seems, as stated at my first report, the mod works for every science experiment available, but "SC-9001 Science Jr" and "Mystery Goo™ Containment Unit".

If you need any other help with some tests, just let me know.

**Comment by dtparr** [comment]
@irineus I attempted to duplicate your bug using KSP 1.4.2.2110 (WindowsPlayer x64) en-us and ForScience 1.5.0, but was unable to do so.  Using your same procedure, all 8 experiments were triggered upon entering the various biomes around KSP.

I tried several attempts, but the only variation was that occasionally upon reverting to launch, the mod would trigger collecting experiments on the runway itself, while it would sometimes require me to leave the runway and return to collect runway experiments.

Given that the experiments you're having issues with are those that require a Scientist on board and that the only difference between our versions is the language, I'm lead to believe that this may be a localization issue.  In particular, though I can't find confirmation in the API docs, I think that the issue may be that at line 257, `if (kerbal.experienceTrait.Title == "Scientist") return true;`, the Title being returned is now localized to "Cientista", and therefore the mod does not believe a scientist is on board.  During my googling, I found but lost a similar sounding issue on a different mod that appeared to be this same issue.

Again, the docs aren't super helpful, but checking other mods, it looks like the localization-independent way to do that is either:
`if (kerbal.protoCrewMember.experienceTrait.Config.Name == "Scientist") return true;`  or 
`if (kerbal.protoCrewMember.experienceTrait.TypeName == "Scientist") return true;`

I'm not setup to compile and test at the moment and I'm loathe to do a dry-coded PR, but hopefully @confusingbits can take that and poke at it a bit to confirm/deny my suspicions.
